tools/packaging/common/istio-iptables.sh

```diff
@@ -59,29 +59,8 @@ function dump {
     ip6tables-save
 }
 
-#
-#  Function check if argument is a valid ipv4 or ipv6 address
-#
-function isValidIP() {
-   if [ "$1" != "${1#*[0-9].[0-9].[0-9].[0-9]}" ]; then
-      true
-   elif [ "$1" != "${1#*:[0-9a-fA-F]}" ]; then
-      true
-   else
-      false
-   fi
-}
-
-#
-# Function return true if agrument is a valid ipv4 address
-#
-function isIPv4() {
-   if [ "$1" != "${1#*[0-9].[0-9].[0-9].[0-9]}" ]; then
-      true
-   else
-      false
-  fi
-}
+# Load ip addresses validation functions
+. ./tools/packaging/common/validations.sh
```

Comment:
Wait, let's not make this complicated. Include it in this file, we don't want 2 files.

Reply:
If you want to make it testable - add another option so iptables can be sourced ( i.e. doesn't do anything ), and include it from the test.

Reply:
@costinm I am sorry but I disagree, modifying "main"application logic with extra parameter just to be able to test it, does not make too much sense to me. Having separate function file allows flexible testing and usage.